scripts/README.me

```diff
@@ -0,0 +1,8 @@
+This directory contains three scripts to:
```

Comment:
was the extension of this intended to be `be` or was a mistake and should be `md`?